src/backend/aspen/cli/db.py

```diff
@@ -408,3 +410,54 @@ def create_phylo_run(
 
         workflow_id = workflow.workflow_id
     print(workflow_id)
+
+
+@db.command("create-mega-fasta")
+@click.option(
+    "public_identifier_input_fh",
+    "--public-identifier-txt",
+    type=click.File("r"),
+    required=True,
+)
+@click.option(
+    "sequences_output_fh", "--sequences-output", type=click.File("w"), required=True
+)
+@click.pass_context
+def create_mega_fasta(
+    ctx, public_identifier_input_fh: io.TextIOBase, sequences_output_fh: io.TextIOBase
+):
+
+    engine = ctx.obj["ENGINE"]
+    session = engine.make_session()
+
+    public_identifiers: Collection[str] = [
```

Comment:
```suggestion
    public_identifiers: Sequence[str] = [
```

sqlalchemy's `in_` works differently than python's `in` sadly.

Reply:
ah interesting, Why Sequence and not Iterable or list type hint here?

Reply:
Generally speaking, variables and parameters should be declared with the lowest type possible (to provide flexibility).  There's a school of thought that return values should be declared with the highest type possible.  I'm not sure I subscribe to that, so happy to debate that one. :)